src/value.cpp

```diff
@@ -856,18 +856,39 @@ namespace Exiv2 {
     }
 
     int LangAltValue::read(const std::string& buf)
-    {
+    {        
         std::string b = buf;
         std::string lang = "x-default";
         if (buf.length() > 5 && buf.substr(0, 5) == "lang=") {
+            const char* ALPLHA = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```

Comment:
I would propose to move these constants to the anonymous namespace or make them static so that we do not instantiate them in every call to the function.

Reply:
Yes, you're right. I've made the change.